core/src/main/java/com/linecorp/armeria/server/AbstractVirtualHostBuilder.java

```diff
@@ -237,6 +241,41 @@ public B service(PathMapping pathMapping,
     }
 
     /**
+     * Binds the specified annotated service object.
```

Comment:
Then maybe `under the path prefix "/"`, which is consistent with the docstring of the versions that take `pathPrefix`. I think it does need some sort of clarification.